activerecord/lib/active_record/migration.rb

```diff
@@ -598,7 +598,7 @@ def call(env)
         @mutex.synchronize do
           @watcher ||= build_watcher do
             @needs_check = true
-            ActiveRecord::Migration.check_pending!(connection)
+            check_pending_migrations_unsafely
```

Comment:
I'd prefer to just call this `check_pending_migrations` rather than have the `unsafely` 😄 (especially since it's an internal method).